source/guides/applications/rake.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Lotus - Guides - Applications Rake Tasks
+---
+
+# Rake Tasks
+
+Lotus ships with default Rake tasks that can be used as _prerequisites_ by developers to build their own tasks.
+
+```shell
+% bundle exec rake -T
+rake environment  # Load the full project
+rake preload      # Preload project configuration
+rake test         # Run tests
+```
+
+## Environment
+
+This Rake task:
+
+  1. Executes all `preload` steps
+  2. Load all the project code
+
+Use this as a Rake task prerequisite when you **DO** need to access to project code (eg. entites, actions, views, etc..)
+
+### Example
+
+Imagine we want to build a Rake task that is able to access project code (eg. a repository)
+
+```ruby
+# Rakefile
+
+task clear_users: :environment do
```

Comment:
should we state clearly that it would load _all_? I think users might be wondering if `environment` here could mean a specific env of an application or not